README.md

```diff
@@ -1,5 +1,32 @@
 # Installation
+
+## Prepare docker
+
+Install docker
```

Comment:
`Install docker` should be just a link to https://docs.docker.com/install/ , without any explanations